crates/runc-shim/src/service.rs

```diff
@@ -144,6 +177,34 @@ impl Shim for Service {
     }
 }
 
+/// Sets the OOM score for the process to the parents OOM score + 1
```

Comment:
All these functions may be grouped as a submodule, so we could reduce conditional compilation.

Reply:
You're right. Should I mark `cgroup` as a new custom feature to replace `target_os = "linux"` at the same time?